app/search/search.component.ts

```diff
@@ -113,6 +117,11 @@ export class SearchComponent implements OnInit {
     }
   }
 
+  download(term: string): void {
```

Comment:
looks like `term` parameter is not used?

Reply:
true
i should add it to the csv somewhere